src/wide/f32x8_t.rs

```diff
@@ -91,6 +101,13 @@ impl f32x8 {
                 )
             } else if #[cfg(all(feature = "simd", target_feature = "simd128"))] {
                 Self(f32x4_eq(self.0, rhs.0), f32x4_eq(self.1, rhs.1))
+            } else if #[cfg(all(feature = "simd", target_arch = "aarch64", target_feature = "neon"))] {
+                unsafe {
+                    Self(
+                        core::mem::transmute(vceqq_f32(self.0, rhs.0)),
```

Comment:
Why do we need transmute here?

Reply:
Because your methods here are "weird" in that they do a comparison but return a floating point type, even though they really are masks. On the other architectures we got "lucky" in that they don't differentiate the types.

Reply:
Well, this is definitely Skia being "weird". And seems like they use [vreinterpretq_f32_u32](https://github.com/google/skia/blob/1f193df9b393d50da39570dab77a0bb5d28ec8ef/include/private/SkNx_neon.h#L204) instead. Should we as well? Not sure how different it from `transmute`.

Reply:
Though I believe I added Neon types to bytemuck, so we could just use bytemuck here.